static/js/ProfilePic.jsx

```diff
@@ -106,6 +106,7 @@ export class ProfilePic extends Component {
                     src={this.state.fileToCropSrc}
                     onClose={this.onCloseCropper}
                     onSave={this.upload}
+                    widthHeightRatio={1}
```

Comment:
interesting that you chose to make it general even though currently the value used is still 1:

Reply:
In TopicEditor.jsx I set the value to 4/3 so it needs to be a prop.